templates/auth.sh.j2

```diff
@@ -1,15 +1,24 @@
 #!/bin/bash
 
-cqlsh -e 'SELECT now() FROM system.local;' -u {{ cassandra_cluster_username }} -p {{ cassandra_cluster_password }} {{ cassandra_address }} {{ cassandra_port }} &>/dev/null
+#is it compatible? returns syntax error!
+
+docker run -t cassandra:{{ cassandra_version }} sh -c "exec cqlsh -e \"SELECT now() FROM system.local;\" -u {{ cassandra_cluster_username }} -p {{ cassandra_cluster_password }} {{ cassandra_address }} {{ cassandra_port }}"
```

Comment:
Install cqlsh on one host and run from there. Alternatively create one-time bootstraping container and run it once.